src/patternfly/demos/Masthead/templates/masthead-template.hbs

```diff
@@ -1,6 +1,13 @@
-{{> page-template-skip-to-content}}
-{{#> masthead masthead--id=(concat page--id '-masthead')}}
-  {{> masthead-toggle}}
+{{!--
```

Comment:
fwiw with the other page demo templates, I put them in the page component dir so we can keep the demo stuff separate from the example stuff and it makes it easier to find files. Though I don't feel strongly about it if you think there is a better way.

An aside, I'd like to organize all of that better once we can make hbs changes in subdirs and the file watcher reload the workspace properly.